src/commonMain/kotlin/ch/dreipol/dreimultiplatform/reduxkotlin/navigation/NavigationReducer.kt

```diff
@@ -1,13 +1,4 @@
 package ch.dreipol.dreimultiplatform.reduxkotlin.navigation
 
-fun navigateBack(state: NavigationState): NavigationState {
-    if (state.sheet != null) {
```

Comment:
@lailabecker funktioniert hier nachher noch der Hardware-Back-Button wenn wir auf einem Sheet sind? Also wollen wir jetzt ein Sheet genau gleich wie ein Screen behandeln?

Reply:
Das sheet ist neu einfach nicht mehr standardmässig dabei, weil wir es nicht in allen Apps gebrauchen. Bei Mammut gibt es das z.B. und dann funktioniert die Logik weiterhin so wie bisher.